django/db/models/base.py

```diff
@@ -44,6 +45,11 @@ def __str__(self):
 DEFERRED = Deferred()
 
 
+@contextmanager
+def noop_cm():
+    yield
```

Comment:
Python 3.7 provides `nullcontext` so I suggest this be at least called `nullcontext` if not placed in a backportsy place so it can be dropped when we support only 3.7+

https://docs.python.org/3/library/contextlib.html#contextlib.nullcontext

Reply:
Good point.